cloudinit/stages.py

```diff
@@ -956,6 +956,8 @@ def apply_network_config(self, bring_up):
         Find the config, determine whether to apply it, apply it via
         the distro, and optionally bring it up
         """
+        from cloudinit.config.schema import validate_cloudconfig_schema
```

Comment:
Yes this is a circular import due to cloudinit.config.schema's `from cloudinit.cmd.devel import read_cfg_paths` which calls `from cloudinit.stages import Init`. I suppose this could be fixed in cloudinit.cmd.devel.__init__ to pull the cloudinit.stages import down into `read_cfg_files` scope instead to avoid exposing the circular import as an alternative.

Reply:
It's fine. I just wanted to make sure.